dist/gmusic.js

```diff
@@ -165,6 +165,15 @@ GMusic.Playback = {
   ALL_SHUFFLE: 'ALL_SHUFFLE',
```

Comment:
We shouldn't remove these constants. Shuffle modes are list of possible modes but not the values themselves. As a programmer, I want to reference `gmusic.setShuffle(GMusic.Playback.ALL_SHUFFLE)` not `gmusic.setShuffle(GMusic.Playback.SHUFFLE_MODES[0])`.